src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -354,7 +354,6 @@ class NBDServer
     void* entry() override
     {
       (server.*func)();
-      server.shutdown();
```

Comment:
BTW, are you aware about `../qa/workunits/rbd/rbd-nbd.sh`? It provides some basic testing. Make sure to add `$CEPH/build/bin` to your `PATH` before running this.